pilot/pkg/serviceregistry/serviceentry/conversion.go

```diff
@@ -194,13 +194,13 @@ func convertServices(cfg config.Config) []*model.Service {
 	}
 
 	out = append(out, buildServices(hostAddresses, cfg.Namespace, svcPorts, serviceEntry.Location, resolution,
```

Comment:
if the method has 9 params - maybe better to refactor a bit - like pass 'cfg' as one param, and serviceEntry as another.